### PR TITLE
Modify cron temporarily

### PR DESCRIPTION
**Pull request by LiangquanLi930**
This is a temporary PR to trigger profiles that are not currently running in 4.12.
When our goal is complete, then we will roll back

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly.yaml [hunk 1]
```diff
@@ -864,7 +864,7 @@ tests:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-baselinecaps-v412
 - as: azure-ipi-baselinecaps-v412-p3-f28-destructive
-  cron: 16 18 2 * *
+  cron: 16 18 4 * *
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly.yaml:867]: Isn't it already executed ?

Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly.yaml:867]: But executed failed, so re-trigger again

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly.yaml:867]: We should use rebuild in that case.

Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly.yaml]: okay, got it, thanks, updated

**Comment by liangxia** [comment]
Unfortunately, conflicts at the last minute

**Comment by LiangquanLi930** [comment]
> Unfortunately, conflicts at the last minute

yes, rebase completed

**Pull request merged**